### PR TITLE
[zh-cn] Resync port-forward-access-application-cluster.md

### PR DESCRIPTION
**Pull request by 0xff-dev**
[zh] add min-kubernetes-server-version
now https://kubernetes.io/zh-cn/docs/tasks/access-application-cluster/port-forward-access-application-cluster/
preview https://deploy-preview-35585--kubernetes-io-main-staging.netlify.app/

### CODE REVIEW

**Comment by windsonsea** [comment]
Could you go through the topic which need to improve further?

**Comment by 0xff-dev** [comment]
> Could you go through the topic which need to improve further?

For the changed files, I have executed the following command, no changes are output.

image

I read this file and found nothing that needs to be changed. If you think there is something that needs to be changed, please help me and I will make some changes.

**Comment by windsonsea** [comment]
Like line 20:
```
本文展示如何使用 `kubectl port-forward` 连接到在 Kubernetes 集群中
运行的 MongoDB 服务。这种类型的连接对数据库调试很有用。
```
We can remove the unnecessary space between zh-cn characters and modify it to:
```
本文展示如何使用 `kubectl port-forward` 连接到在 Kubernetes 集群中运行的 MongoDB 服务。
这种类型的连接对数据库调试很有用。
```
 --- 
Like line 41:
```
1. 创建一个运行 MongoDB 的 deployment：
```
If an en word `deployment` is localized to zh-cn, it will be changed to:
```
1. 创建一个运行 MongoDB 的 Deployment：
```

**Comment by 0xff-dev** [comment]
@windsonsea Please review again if you have time, thanks.

File: content/zh-cn/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 1]
```diff
@@ -149,7 +151,7 @@ for database debugging.
 <!--
 3. Verify that the MongoDB server is running in the Pod, and listening on port 27017:
 -->
-3. 验证 MongoDB 服务是否运行在 Pod 中并且在 27017 端口上监听：
+3. 验证 Pod 中的 MongoDB 服务是否在运行并且监听 27017 端口：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:154]:
```suggestion
3. 验证 MongoDB 服务是否运行在 Pod 中并且在监听 27017 端口：
```
The original text is better for the first half in my opinion.

**Comment by Sea-n** [comment]
Looks good to me, just changing the PR title to be more clear.

/retitle [zh-cn] Resync port-forward-access-application-cluster.md

**Pull request merged**